virtcontainers/cni.go

```diff
@@ -155,14 +155,18 @@ func (n *cni) add(sandbox *Sandbox, config NetworkConfig, netNsPath string, netN
 
 // remove unbridges and deletes TAP interfaces. It also removes virtual network
```

Comment:
While we are in here, should we update the function description to be more accurate (ie, handle the non-virtual case as well).

Reply:
Fixed.